estate/models/estate_property.py

```diff
@@ -0,0 +1,126 @@
+from odoo import models, fields, api
```

Comment:
import sequence should be alphabetical wise 
please update the all import for the same
```suggestion
from odoo import api, fields, model
```